api/saml/configuration/model.py

```diff
@@ -64,6 +64,54 @@ class SAMLConfiguration(ConfigurationGrouping):
         format="narrow",
     )
 
+    patron_id_use_name_id = ConfigurationMetadata(
```

Comment:
Would you like to have all the attributes of `SAMLConfiguration` to be capitalized? If yes, then probably, I would also need to do the same for ProQuest configuration classes: [ProQuestAPIClientConfiguration](https://github.com/NYPL-Simplified/circulation/blob/develop/api/proquest/client.py#L21), and [ProQuestOPDS2ImporterConfiguration](https://github.com/NYPL-Simplified/circulation/blob/develop/api/proquest/importer.py#L85).

Reply:
@jonathangreen @leonardr Do you have thoughts on this?

Reply:
I don't have a strong opinion. Those are instances of application classes, so not your typical constants. But they are being used as groupings of constants.

Since we don't currently use `ConfigurationMetadata` everywhere I think it makes sense to use lowercase for these configuration objects at least so long as we're using all caps for the names of configuration _settings_. That way it's clear that `patron_id_use_name_id` is a configuration object and `ALLOW_HOLDS` is just a string.